doc/src/programmers-guide.md

```diff
@@ -1804,12 +1804,14 @@ Currently, the following options are supported:
 |------------|--------------|-------------|
 | `{socket, reuseaddr}` | `boolean()` | Sets `SO_REUSEADDR` on the socket. |
 | `{socket, linger}` | `#{onoff => boolean(), linger => non_neg_integer()}` | Sets `SO_LINGER` on the socekt. |
+| `{otp, rcvbuf}` | `non_neg_integer()` | Sets the default buffer size (in bytes) on receive calls.  This value is only used if the `Length` parameter of the `socket:recv` family of functions has the value `0`; otherwise, the specified non-zero length in the `socket:recv` takes precendence. |
```

Comment:
Can we have 0 for this value?
OTP specifies `default | integer()>0`

Reply:
Added note to table and TODO in the code.  Not sure why CI did not kick off.